platforms/android/tangram/src/main/java/com/mapzen/tangram/networking/DefaultHttpHandler.java

```diff
@@ -22,63 +22,40 @@
 /**
  * {@code DefaultHttpHandler} is an implementation of {@link HttpHandler} using OkHTTP.
  * Customize this class for your own application by subclassing and overriding
- * {@link DefaultHttpHandler#configureClient(OkHttpClient.Builder)} and
  * {@link DefaultHttpHandler#configureRequest(HttpUrl, Request.Builder)}.
  */
 public class DefaultHttpHandler implements HttpHandler {
 
     private OkHttpClient okClient;
 
     /**
-     * Construct an {@code DefaultHttpHandler} with default options.
+     * @return OkHttp client builder with recommended settings for Tangram.
      */
-    public DefaultHttpHandler() {
-        final OkHttpClient.Builder builder = new OkHttpClient.Builder()
+    public static OkHttpClient.Builder GetClientBuilder() {
```

Comment:
As far as I know, the naming convention for Java is that methods should never start with an uppercase letter (also not static methods). I suggest to call the method `createClientBuilder` (or `createDefaultClientBuilder`)

Reply:
Oops! Good catch - I've been writing too much C# recently